javav2/example_code/ses/src/main/java/com/example/sesv2/SendwithHeader.java

```diff
@@ -0,0 +1,116 @@
+
+/**
+ * Before running this AWS SDK for Java (v2) example, set up your development
+ * environment, including your credentials.
+ * <p>
+ * For more information, see the following documentation topic:
+ * <p>
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+
+// snippet-start:[ses.java2.send.header.sesv2.main]
+public class SendwithHeader {
+
+    public static void main(String[] args) {
+        final String usage = """
+                             
+            Usage:
+                <sender> <recipient> <subject>\s
+                             
+            Where:
+                sender - An email address that represents the sender.\s
+                recipient - An email address that represents the recipient.\s
+                subject - The subject line.\s
+            """;
+
+        if (args.length != 3) {
+            System.out.println(usage);
+            System.exit(1);
+        }
+
+        String sender = args[0];
+        String recipient = args[1];
+        String subject = args[2];
+        Region region = Region.US_EAST_1;
+        SesV2Client sesv2Client = SesV2Client.builder()
+                .region(region)
+                .build();
+
+        String bodyHTML = """
+                <html>
+                    <head></head>
+                    <body>
+                        <h1>Hello!</h1>
+                        <p>See the list of customers.</p>
+                    </body>
+                </html>
+                """;
+
+        sendWithHeader(sesv2Client, sender, recipient, subject, bodyHTML);
+        sesv2Client.close();
+    }
+
+    /**
+     * Sends an email using the AWS SES V2 client.
+     *
+     * @param sesv2Client the SES V2 client to use for sending the email
+     * @param sender the email address of the sender
+     * @param recipient the email address of the recipient
+     * @param subject the subject of the email
+     * @param bodyHTML the HTML content of the email body
+     */
+    public static void sendWithHeader(SesV2Client sesv2Client,
+                                      String sender,
+                                      String recipient,
+                                      String subject,
+                                      String bodyHTML) {
+        EmailContent emailContent = EmailContent.builder()
+                .simple(Message.builder()
+                        .body(b -> b.html(c -> c.charset(UTF_8.name()).data(bodyHTML))
+                                .text(c -> c.charset(UTF_8.name()).data(bodyHTML)))
+                        .subject(c -> c.charset(UTF_8.name()).data(subject))
+                        .headers(List.of(
+                                MessageHeader.builder()
+                                        .name("List-Unsubscribe")
+                                        .value("<https://nutrition.co/?address=x&topic=x>, <mailto:unsubscribe@nutrition.co?subject=TopicUnsubscribe>")
```

Comment:
I wonder if you should anonymize this more? nutrition.co doesn't exist, now, but it might.

Reply:
I think this code is ok